internal/cmd/plugin/status/status.go

```diff
@@ -930,6 +930,13 @@ func (fullStatus *PostgresqlStatus) printTablespacesStatus() {
 		headerColor = aurora.Yellow
 	}
 
+	temporaryTablespaces := stringset.New()
+	for _, tablespace := range fullStatus.Cluster.Spec.Tablespaces {
+		if tablespace.Temporary {
+			temporaryTablespaces.Put(tablespace.Name)
+		}
+	}
+
```

Comment:
this bit here seems more complex than needed. Below we can simply check for .Temporary == true in the `Temporary` column.